package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
Specifying `dist/redbox` and the format as `cjs` was the only way I could make this work.  I don't remember the error messages.  I can try to reproduce, step-by-step, all the errors I ran into when trying to install and use these 3 packages.

Actually, I went ahead and put together a repro repo: https://github.com/timbur/another-systemjs-test

How I reached that point:

1.  `npm install express --save`
2.  `npm install jspm --save-dev`
3.  `jspm init` -> Babel -> etc.
4.  `jspm install npm:babel-plugin-react-transform npm:react@^0.14.0-rc1 npm:react-dom npm:react-transform-catch-errors npm:redbox-react`
5.  Create `config.dev.js` and `reporterOptions.js` and add `transform` to `babelOptions`.
6.  Create `index.html` and `server.js` using `express` static files.
7.  Create `mount.js` and `src/components/App.js`.
8.  Open http://localhost:8080 and get an error that eventually boils down to the double interop, or at least that's what I think it is.  Maybe I'm just being stupid.  But ideally, I think everything I've done here should have worked without any extra effort or debugging.